System/OsPath/Data/ByteString/Short/Internal.hs

```diff
@@ -441,3 +449,29 @@ errorEmptySBS fun = moduleError fun "empty ShortByteString"
 moduleError :: HasCallStack => String -> String -> a
 moduleError fun msg = error (moduleErrorMsg fun msg)
 {-# NOINLINE moduleError #-}
+
+compareByteArraysOff :: BA  -- ^ array 1
```

Comment:
Unfortunately, bytestring doesn't expose this internal and we need to to be compatible with GHC-8.2.2 and lower.